README.md

```diff
@@ -73,6 +73,31 @@ cd gitleaks
 make build
 ```
 
+### Load Configuration
```

Comment:
@bufferoverflow this section should go after the Pre-commit section and before the Configuration section

Reply:
@zricethezav I just changed accordingly with a9074dda105117004476de516f11e29212436c71